### PR TITLE
PT-2561: Data import UI fixes

### PR DESCRIPTION
**Pull request by ErickLipin**
https://virtocommerce.atlassian.net/browse/PT-638

### CODE REVIEW

File: src/VirtoCommerce.Platform.Web/wwwroot/Localizations/en.VirtoCommerce.Platform.json [hunk 1]
```diff
@@ -177,8 +177,11 @@
                 "title": "Data import",
                 "labels": {
                     "start-import": "Start import",
-                    "select-file": "Select .zip file to import. The file must be produced by Commerce Manager data export.",
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Web/wwwroot/Localizations/en.VirtoCommerce.Platform.json:180]: Why did you remove localization?

File: src/VirtoCommerce.Platform.Web/wwwroot/Localizations/ru.VirtoCommerce.Platform.json [hunk 2]
```diff
@@ -155,8 +155,11 @@
                 "title": "Импорт данных",
                 "labels": {
                     "start-import": "Начать импорт",
-                    "select-file": "Выберите файл .zip для импорта. Файл должен быть создан путем экспорта данных Commerce Manager.",
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Web/wwwroot/Localizations/ru.VirtoCommerce.Platform.json:158]: Same.

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js [hunk 3]
```diff
@@ -1,125 +1,129 @@
...
+        blade.updatePermission = 'platform:exportImport:import';
+        blade.headIcon = 'fa fa-download';
+        blade.title = 'platform.blades.import-main.title';
+        blade.isLoading = false;
+        $scope.importRequest = {};
+        var origManifest;
+
+        $scope.$on("new-notification-event", function (event, notification) {
+            if (notification.jobId && notification.finished) {
+                blade.toolbarCommands.splice(0, 1, commandClose);
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js:13]: 1. The useless operation, look on Line 33, did you suppose to `return`?
2. Extract method.

**Review by Vectorfield4 — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.Platform.Web/wwwroot/Localizations/en.VirtoCommerce.Platform.json (hunk 1 above)
Comment (eMazeika) [reply src/VirtoCommerce.Platform.Web/wwwroot/Localizations/en.VirtoCommerce.Platform.json:180]: ?

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js [hunk 4]
```diff
@@ -1,125 +1,129 @@
...
-        },
-        executeMethod: function () {
-            exportImportResourse.taskCancel({ jobId: blade.notification.jobId }, null, function (data) {
-            });
+
+        $scope.startProcess = function () {
+            blade.isLoading = true;
+            exportImportResourse.runImport($scope.importRequest,
+                function (data) { blade.notification = data; blade.isLoading = false; },
+                function (error) { bladeNavigationService.setError('Error ' + error.status, blade); });
```
Comment (eMazeika) [inline src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js:31]: this error callback is duplicating the default functin and should be removed

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js [hunk 5]
```diff
@@ -1,125 +1,129 @@
...
-            fn: function (i /*{File|FileLikeObject}*/, options) {
-                return i.name.toLowerCase().endsWith('.zip');
+        var commandCancel = {
+            name: 'platform.commands.cancel',
+            icon: 'fa fa-times',
+            canExecuteMethod: function () {
+                return blade.notification && !blade.notification.finished;
+            },
+            executeMethod: function () {
+                exportImportResourse.taskCancel({ jobId: blade.notification.jobId }, null, function (data) {
```
Comment (eMazeika) [inline src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js:43]: remove empty function

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.tpl.html [hunk 6]
```diff
@@ -39,59 +33,110 @@
                     <div class="list-descr">{{importRequest.exportManifest.platformVersion}}</div>
                 </li>
             </ul>
-            
+            <!--  -->
```
Comment (eMazeika) [inline src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.tpl.html:36]: remove empty

**Review by eMazeika — CHANGES_REQUESTED** [review]
add screenshot / gif / video to visual PRs like this

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js (hunk 3 above)
Comment (ErickLipin) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js:13]: Git was layer this.

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.tpl.html (hunk 6 above)
Comment (ErickLipin) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.tpl.html]: done

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js (hunk 5 above)
Comment (ErickLipin) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js]: done

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js (hunk 4 above)
Comment (ErickLipin) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/blades/import-main.js]: removed

File: src/VirtoCommerce.Platform.Web/wwwroot/Localizations/ru.VirtoCommerce.Platform.json (hunk 2 above)
Comment (ErickLipin) [reply src/VirtoCommerce.Platform.Web/wwwroot/Localizations/ru.VirtoCommerce.Platform.json:158]: Returned

File: src/VirtoCommerce.Platform.Web/wwwroot/Localizations/en.VirtoCommerce.Platform.json (hunk 1 above)
Comment (ErickLipin) [reply src/VirtoCommerce.Platform.Web/wwwroot/Localizations/en.VirtoCommerce.Platform.json:180]: Returned
